nxt_editor/integration/__init__.py

```diff
@@ -32,29 +33,41 @@ def _safe_import_package(package_name, global_name=None):
 
     def _install_and_import_package(self, module_name, package_name=None,
                                     global_name=None):
+        """Calls a subprocess to pip install the given package name and then
+        attempts to import the new package.
+
+        :param module_name: Desired module to import after install
+        :param package_name: pip package name
+        :param global_name: Global name to access the module if different
+        than the module name.
+        :raises: subprocess.CalledProcessError
+        :return: bool
+        """
         if package_name is None:
             package_name = module_name
         if global_name is None:
             global_name = module_name
-        args = ['install', package_name]
-        self.ensure_pip()
-        import pip
-        if hasattr(pip, 'main'):
-            pip.main(['install', package_name])
-        else:
-            pip._internal.main(['install', package_name])
+        environ_copy = dict(os.environ)
+        environ_copy["PYTHONNOUSERSITE"] = "1"
```

Comment:
For this(and the rest of the pip calls), what does PYTHONUSERSITE do? I'm familiar with `--user` as a flag to pip install, but I've never used this environment variable, and unfortunately the internet didn't have a quick explanation.

Reply:
Got the info from [here](https://github.com/robertguetzkow/blender-python-examples/blob/1bbeb44488cb775b15acea1230585a93cd04584a/add_ons/install_dependencies/install_dependencies.py#L106). Maybe its not a good idea for other DCCs? The worst that will happen is a dependency will be installed twice.